### PR TITLE
[13.0][MIG] sale_order_invoicing_queued: Migration to 13.0

### PR DESCRIPTION
**Pull request by pedrobaeza**
* Standard procedure
* README updated
* Manifest version updated
* Adjusted code to v13

cc @Tecnativa TT20449

### CODE REVIEW

**Review by carlosdauden — APPROVED** [review]
Code LGTM

File: sale_order_invoicing_queued/models/sale_order.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Copyright 2019 Tecnativa - Pedro M. Baeza
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+try:  # The try can be removed in v12 as not needed
```
Comment (sergio-teruel) [inline sale_order_invoicing_queued/models/sale_order.py]: Remove

**Review by sergio-teruel — APPROVED** [review]

File: sale_order_invoicing_queued/models/sale_order.py (hunk 1 above)
Comment (pedrobaeza) [reply sale_order_invoicing_queued/models/sale_order.py]: Thanks, removed!

**Pull request merged**